modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -48,17 +48,17 @@ definition {
 		}
 
 		task ("When the administrator disables a feature flag") {
-			Check.checkToggleSwitch(
+			ClickNoError(
 				locator1 = "FeatureFlag#TOGGLE_ROW",
 				title_row = "Terms and Conditions pages",
 				toggle_state = "Enabled");
 		}
 
 		task ("Then assert the toggle is disabled state") {
-			AssertElementPresent(
+			AssertElementNotPresent(
 				locator1 = "FeatureFlag#TOGGLE_ROW",
 				title_row = "Terms and Conditions pages",
-				toggle_state = "Disabled");
+				toggle_state = "Enabled");
```

Comment:
Why is this change needed?

Reply:
if it's a matter of fitting with the task better, I would say it was better before. Since we're asserting it's Disabled state